python-sdk/src/astro/options.py

```diff
@@ -57,10 +57,16 @@ class SnowflakeLoadOptions(LoadOptions):
         https://docs.snowflake.com/en/sql-reference/sql/copy-into-table.html#format-type-options-formattypeoptions
     :param copy_options: Specify one or more of the copy option as key-value pair. Read more at:
         https://docs.snowflake.com/en/sql-reference/sql/copy-into-table.html#copy-options-copyoptions
+    :param storage_integration: Specify the previously created Snowflake storage integration
+    :param validation_mode: Specify the supported validation mode; RETURN_n_ROWS | RETURN_ERRORS | RETURN_ALL_ERRORS
+        This instructs the COPY command to validate the data files instead of loading them into the specified table;
+        i.e. the COPY command tests the files for errors but does not load them. Read more at:
+        https://docs.snowflake.com/en/sql-reference/sql/copy-into-table.html#optional-parameters
     """
 
     file_options: dict = attr.field(init=True, factory=dict)
     copy_options: dict = attr.field(init=True, factory=dict)
+    validation_mode: Optional[str] = attr.field(default=None)
```

Comment:
isn't this part of copy_options only?

Reply:
No this is not part of `copy_options`. `validation_mode` is a separate parameter which has to be passed. Read more at: https://docs.snowflake.com/en/sql-reference/sql/copy-into-table.html#optional-parameters